wiki/Game_mode/de.md

```diff
@@ -1,25 +1,22 @@
----
-outdated_translation: true
-outdated_since: 422698dbf36efb2be8537ab4d319428032966405
----
-
 # Spielmodi
 
-osu! hat vier **Spielmodi** mit verschiedenen Mechaniken, die ursprünglich von anderen Rhythmusspielen inspiriert sind:
+osu! hat vier integrierte **Spielmodi** mit verschiedenen Mechaniken, die ursprünglich von anderen Rhythmusspielen inspiriert sind:
```

Comment:
integriert doesn't seem as intuitive as built-in..

`mitgelieferte` or something maybe?

Reply:
```suggestion
osu! hat vier mitgelieferte **Spielmodi** mit verschiedenen Mechaniken, die ursprünglich von anderen Rhythmusspielen inspiriert sind:
```

Reply:
`beigefügt` maybe?

Reply:
hmm i prefer `integriert` over beigefügt. `eingebaut` is also an option but meh.. not sure

Reply:
```suggestion
osu! hat vier integrierte **Spielmodi** mit verschiedenen Mechaniken, die ursprünglich von anderen Rhythmusspielen inspiriert sind:
```